packages/babel-traverse/test/fixtures/scope/remove-nested-let-path/plugin.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function () {
+  return {
+    visitor: {
+      VariableDeclarator(path) {
+        switch (path + "") {
+          case "a = 33":
+            path.remove();
+
+          case "a = 42":
+            expect(path.scope.getBinding("a")).toBeDefined();
```

Comment:
If this line never runs it will not fail. We should push a comment in the output saying `"// a is defined? " + path.scope.getBinding("a")`, so that if the comment is missing the test fails.

Also, could you use an if to make this slightly easier to follow?
```js
const { value } = path.node.init;
if (value === 33) path.remove();
t.addComment(
  path.getProgramParent().path.node,
  "trailing",
  `// a is defined when visiting a=${value}? ${path.scope.hasBinding("a")}` 
);
``